### PR TITLE
Add links for OpenLink Virtuoso, on par with Apache Jena TDB

### PR DESCRIPTION
**Pull request by TallTed**

### CODE REVIEW

**Comment by Ostrzyciel** [comment]
Hi, sorry, I have somehow missed this pull request! :(

Thank you for the PR. Good point about giving more examples, but I wouldn't want this to look like an ad for Virtuoso... so maybe just have one variation of Virtuoso mentioned. I think it's fair, given that Jena TDB2 is free software and there is no company behind it.

My suggestion is to go with something like: 

> In this task, an RDF store is set up (for example, [Apache Jena TDB2](https://jena.apache.org/documentation/tdb2/index.html), [OpenLink Virtuoso Open Source](https://github.com/openlink/virtuoso-opensource/), or [RDF4J's NativeStore](https://rdf4j.org/documentation/programming/repository/)) and then loaded with a flat dump of RDF statements (triples or quads).

What do you think?

File: docs/tasks/flat-rdf-store-loading/index.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ Flat distributions of any dataset in the [`flat` category](../../categories/flat
 
 ### Workload
 
-In this task, an RDF store is set up (for example, [Apache Jena TDB2](https://jena.apache.org/documentation/tdb2/index.html) or Virtuoso) and then loaded with a flat dump of RDF statements (triples or quads).
+In this task, an RDF store is set up (for example, [Apache Jena TDB2](https://jena.apache.org/documentation/tdb2/index.html), [OpenLink Virtuoso Open Source v7.x](https://github.com/openlink/virtuoso-opensource/), or [OpenLink Virtuoso Enterprise Edition v8.x](https://shop.openlinksw.com/license_generator/virtuoso-download/)) and then loaded with a flat dump of RDF statements (triples or quads).
```
Comment (TallTed) [inline docs/tasks/flat-rdf-store-loading/index.md:13]: Applying suggestion from @Ostrzyciel 
```suggestion
In this task, an RDF store is set up (for example, [Apache Jena TDB2](https://jena.apache.org/documentation/tdb2/index.html), [OpenLink Virtuoso Open Source v7.x](https://github.com/openlink/virtuoso-opensource/), or [RDF4J's NativeStore](https://rdf4j.org/documentation/programming/repository/)) and then loaded with a flat dump of RDF statements (triples or quads).
```

**Comment by TallTed** [comment]
Back to you, @Ostrzyciel!

**Comment by Ostrzyciel** [comment]
Ok, done. I moved this to a new PR in the correct repo, where the task definition actually lives: https://github.com/RiverBench/category-flat/pull/6 I hope you don't mind :)

This repo (riverbench.github.io) only hosts generated documentation pages, so changes here would get overwritten automatically. To be honest, this is poorly documented. This should be improved in the next release with this task: https://github.com/RiverBench/RiverBench/issues/53

Anyway, closing.

**Pull request closed without merging**